doc/nrf/libraries/networking/multicell_location.rst

```diff
@@ -35,19 +35,19 @@ To use the location services, see the respective documentation for account setup
    nRF9160 DK and Thingy:91 devices are shipped with RSA256 certificates.
    To start using the Multicell location library with nRF Cloud, you must perform either of the following actions:
 
-      * Delete the device from nRF Cloud and reprovision it with a new ES256 device certificate. See :ref:`nrf9160_ug_updating_cloud_certificate` for more information.
-      * Register a separate key for JWT signing as described in `Securely Generating Credentials on the nRF9160`_.
+   * Delete the device from nRF Cloud and reprovision it with a new ES256 device certificate. See :ref:`nrf9160_ug_updating_cloud_certificate` for more information.
+   * Register a separate key for JWT signing as described in `Securely Generating Credentials on the nRF9160`_.
 
 .. reprovision_cert_note_end
 
 The required credentials for the location services are configurable using Kconfig options.
 There is no difference in the API calls when using the services.
 
-The library has an API to handle provisioning of the required TLS certificates for the location services, :c:func:`multicell_location_provision_certificate`.
+The library uses the :c:func:`multicell_location_provision_certificate` function to handle provisioning of the required TLS certificates for the location services.
```

Comment:
The meaning changes. Library doesn't use this function that is in its own API. The client of the multicell location library uses this function.

Reply:
Roger that, let's ignore my change.